src/js/translations/en-US.json

```diff
@@ -21,7 +21,7 @@
   "DASHBOARD.PANEL_HEADING.CPU": "CPU Allocation",
   "DASHBOARD.PANEL_HEADING.MEMORY": "Memory Allocation",
   "DASHBOARD.PANEL_HEADING.DISK": "Disk Allocation",
-  "DASHBOARD.PANEL_HEADING.SERVICES_HEALTH": "Services Health",
+  "DASHBOARD.PANEL_HEADING.SERVICES_HEALTH": "Services Status",
```

Comment:
Rename this constant so it accurately represents the information it's now showing?

Reply:
I don't get it? the information is the service status.

Reply:
I mean the key name: "SERVICES_HEALTH" could be renamed to "SERVICES_STATUS"